src/core/packages/ui-settings/server-internal/src/saved_objects/migrations.test.ts

```diff
@@ -416,3 +416,38 @@ describe('ui_settings 8.9.0 migrations', () => {
     });
   });
 });
+
+describe('ui_settings 8.19.0 migrations', () => {
```

Comment:
See other comment, this needs to be a model version migration (which is fully decoupled from stack versions).